testsuite/integration/basic/src/test/config/arq/arquillian.xml

```diff
@@ -7,7 +7,7 @@
     <container qualifier="jboss" default="true">
         <configuration>
             <property name="jbossHome">${basedir}/target/wildfly</property>
-            <property name="javaVmArguments">${server.jvm.args}</property>
+            <property name="javaVmArguments">${server.jvm.args} -agentlib:jdwp=transport=dt_socket,address=12345,server=y,suspend=y</property>
```

Comment:
This looks like it was left here my mistake.

Reply:
Removed.